src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/resource/prison/UserController.kt

```diff
@@ -471,10 +563,30 @@ data class PrisonStaffUserDto(
           lastName,
           status,
           primaryEmail,
-          generalAccount,
-          adminAccount,
+          generalAccount?.let { UserCaseloadDto.fromDomain(it) },
+          adminAccount?.let { UserCaseloadDto.fromDomain(it) },
         )
       }
     }
   }
 }
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Schema(description = "User & Caseload Information")
+data class UserCaseloadDto(
+  @Schema(description = "User name", example = "John1", required = true) val username: String,
+  @Schema(description = "Indicates that the user is active", example = "true", required = true) val active: Boolean,
+  @Schema(description = "Type of user account", example = "GENERAL", required = true) val accountType: PrisonUsageType = PrisonUsageType.GENERAL,
+  @Schema(description = "Active Caseload of the user", example = "BXI", required = false) val activeCaseload: PrisonCaseload?,
```

Comment:
PrisonCaseload is also not a view model class. We should use a view model equivalent of this here (with Swagger annotations). Also on the line below.